src/service/video/docking.js

```diff
@@ -1042,16 +1158,16 @@ export class VideoDocking {
 
   /**
    * @param {!../../video-interface.VideoOrBaseElementDef} video
-   * @param {!RelativeX} posX
-   * @param {!RelativeY} posY
+   * @param {!DockTargetDef} target
    * @param {number} step
    */
-  setCurrentlyDocked_(video, posX, posY, step) {
+  setCurrentlyDocked_(video, target, step) {
     if (!this.isCurrentlyDocked_(video)) {
       this.updateControlsBasedOn_(video.element);
     }
 
-    this.currentlyDocked_ = {video, posX, posY, step};
+    this.currentlyDocked_ = /** @type {!DockedDef} */ (
+      Object.assign(this.currentlyDocked_ || {}, {video, target, step}));
```

Comment:
Is `Object.assign` necessary here? It looks like it will never merge the values, because they are annotated to be  always defined.

Reply:
It was working just fine, but restructured for readability.